.github/workflows/maven.yaml

```diff
@@ -0,0 +1,40 @@
+name: Publish icu4j.jar/utilities.jar to GH Maven
+on:
+  release:
+    types: [created]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: icu4j ant releaseCLDR
+        run: ant -noinput releaseCLDR -f icu4j/build.xml
+      - name: deploy it
+        run: |
+          MYVERSION=$(echo ${GITHUB_REF} | cut -d/ -f3 | tr . _)
+          echo Github Ref ${GITHUB_REF} @ ${GITHUB_SHA}, version ${MYVERSION}
+          mvn deploy:deploy-file -DgroupId=com.ibm.icu \
+            -DartifactId=icu4j \
```

Comment:
If we aren't able to directly use the "vanilla" version of ICU4J directly in CLDR -- meaning that CLDR's ICU4J has any sort of differences that need to be maintained -- then I think it would be cleaner to consider this as a distinct entity in Maven, which means using a distinct value in `artifactId`.  (ex: `cldr-icu4j`).

Extending the version number as is currently configured is certainly one way to do it, too.  I feel like it could be slightly error-prone (complexity on user side to due to required mental overhead) -- see subsequent comment.

Using the [classifier field of a Maven artifact](https://maven.apache.org/plugins/maven-deploy-plugin/examples/deploying-with-classifiers.html) is a third option, which could be useful is you want to keep the connection to ICU4J and treat this as a variant.  This feature isn't commonly used, but maybe it makes sense here if you want to preserve the semantic relationship to ICU4J both in name and in version number.

Reply:
> If we aren't able to directly use the "vanilla" version of ICU4J directly in CLDR -- meaning that CLDR's ICU4J has any sort of differences that need to be maintained 

It's not CLDR's icu4j (no code change), but it is an arbitrary snapshot. We absolutely use the vanilla version of ICU4J in CLDR. However, consider what has to happen right now, September 8th - such as this PR https://github.com/unicode-org/cldr/pull/702/files#diff-42a78bd8fb3858e8442d2c9f16efb057  by @pedberg-icu  - that is the target for this ICU PR.

CLDR and ICU are both revving their versions.  ICU will wait for new CLDR data, and CLDR needs a new ICU that supports the new CLDR features so that CLDR can use that ICU…  yes, interlock.

So what happens in this PR is that an arbitrary snapshot of ICU4J (including the utilities.jar 'extras') is checked into CLDR's repo. Now, at some future time, a release version of ICU4J could be checked in (plus the right version of utilities.jar).  There's no special ICU version, other than than being a special snapshot.

  This build would get kicked off on a 'normal' release tag as well.   There is an accompanying utilities.jar, though, which is not part of a normal build.

So, when ICU 68.1 is released, there will be a `release-68-1` tag and official binaries (uploaded to maven by other means).  But this workflow would also create an icu4j.jar which would be the equivalent of the official 68.1 release (though perhaps unsigned). 

**I could see using the classifier**, to mark all of this stuff as pertaining to CLDR and not intended for general use.  

I'll reply about the version in your next comment.